src/frontend/data/sideEffects/createTimedTextTrack/createTimedTextTrack.ts

```diff
@@ -0,0 +1,38 @@
+import { API_ENDPOINT } from '../../../settings';
+import { modelName } from '../../../types/models';
+import { TimedText, timedTextMode } from '../../../types/tracks';
+import { Nullable } from '../../../utils/types';
+
+/**
+ * Create a new timedtexttrack record for a language-mode combination.
+ * @param jwt The token that will be used to authenticate with the API.
+ * @param language The language for the new timedtexttrack (from the list of available choices).
+ * @param mode The mode for the new timedtexttrack.
+ */
+export const createTimedTextTrack = async (
+  jwt: Nullable<string>,
+  language: string,
+  mode: timedTextMode,
+) => {
+  const response = await fetch(
+    `${API_ENDPOINT}/${modelName.TIMEDTEXTTRACKS}/`,
+    {
+      body: JSON.stringify({ language, mode }),
+      headers: {
+        Authorization: `Bearer ${jwt}`,
```

Comment:
you should maybe throw an error if `jwt` is null ? The API will reject you and you save a call to the server.